### DISCUSSION TITLE
Job availability status in the svg file

### DISCUSSION THREAD

**Discussion by nicokempe** [lowlighter/metrics#1079] [General]
Hey, I don't know if this question stems from a bad view of the configuration files, but I couldn't find anything on the subject in the repository even with the keyword 'jobs'.
The GitHub Jobs project has ended, I was listed as available. However, I am no longer available, I can no longer remove the checkmark in the profile because the feature has been removed. However, the svg file that is created in my profile ReadMe still states that I am available for jobs. Is there a configuration for this, is it some kind of 'bug' left over from the project or what can I do to remove this section from my readme? (Also, thank you for this project, it is reliable and just wonderful)

![github-profile_jobs-in-readme](https://user-images.githubusercontent.com/50241630/170839631-22bad42a-b901-4509-92e6-cf20ae8868b2.png)

**Comment by lowlighter** [comment]
Hi 👋 !

Yes this is shown when the `isHireable` field of a user is set, but like you said it doesn't seem to be editable anymore although the field is still there in database 😕 

I think the best would be to migrate this towards an option in next version, but in the meantime you can workaround this on current one by using [custom css injection](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-custom-css-styling) like below :

```yml
with:
  extras_css: |
    .field.hire {
      display: none;
    }
```

Thanks a lot for supporting this project ❤️ !

**Reply by lowlighter** [reply]
Was shipped in #1081
